src/NServiceBus.Core/Transports/Msmq/CorrelationIdMutatorForBackwardsCompatibilityWithV3.cs

```diff
@@ -1,28 +0,0 @@
-﻿namespace NServiceBus.Transports.Msmq
-{
-    using System;
-    using MessageMutator;
-    using Unicast.Messages;
-
-    [ObsoleteEx(RemoveInVersion ="6.0")]
-    class CorrelationIdMutatorForBackwardsCompatibilityWithV3 : IMutateOutgoingTransportMessages
```

Comment:
@johnsimons sounds reasonable. r there any other changes in this u think should be handled in the same way?